### PR TITLE
Discover movies

### PR DESCRIPTION
**Pull request by JustJakeSeymour**
# Pull request from `discover_movies` to `main`
## Issue addressed:
- Complete Discover Movies (index) page to wireframe and user story spec.
## Proposed changes:
- Add views, routes, & controllers for user_movies, user_discovery
- write tests to confirm discover movies page works as required by spec.
## Reviewer:
- @Pocketzs

### CODE REVIEW

File: spec/features/users/discover_spec.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe "Discover Movies Page" do
+  before :each do
+    @user = create(:user)
+
+    visit user_discover_path(@user)
+  end
+
```
Comment (Pocketzs) [inline spec/features/users/discover_spec.rb]: I think we need to delete this file from your old tests

**Review by Pocketzs — CHANGES_REQUESTED** [review]
Delete discover_spec.rb.  The new version is `discover/index_spec.rb`.  I think it's an old one hanging around

File: spec/features/users/discover_spec.rb (hunk 1 above)
Comment (JustJakeSeymour) [reply spec/features/users/discover_spec.rb]: Great catch

**Review by Pocketzs — APPROVED** [review]
Branch run locally with 100% passing.  Good for setup of the next user stories

**Pull request merged**